pypesto/sampling/__init__.py

```diff
@@ -11,6 +11,6 @@
 from .adaptive_metropolis import AdaptiveMetropolisSampler
 from .parallel_tempering import ParallelTemperingSampler
 from .adaptive_parallel_tempering import AdaptiveParallelTemperingSampler
-from .pymc3 import Pymc3Sampler
 from .result import McmcPtResult
 from .diagnostics import geweke_test
+from .pymc3 import Pymc3Sampler
```

Comment:
i guess this is just style?

Reply:
ah right, that was just from some intermediate mess-up